python/shotgun_fields/image_widget.py

```diff
@@ -279,24 +318,53 @@ def _replace_image(self):
         """
         self._upload_image()
 
-    def _scale_pixmap(self):
+    def _scale_pixmap(self, pixmap):
         """
         Scale the pixmap in preparation for display.
         """
-        scaled_pixmap = self._pixmap.scaled(
+        return pixmap.scaled(
```

Comment:
I believe `KeepAspectRatioByExpanding` will cause part of the image to be clipped in tall/wide scenarios. My expectation is that user's would prefer to see the whole image, but I'm happy to switch if it should clip. :)